lab_1/main.py

```diff
@@ -49,7 +82,13 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top = []
```

Comment:
Do you really need to define it?

Reply:
Yes, cause if not I will get None as a result when isinstance(freq_dict, dict) is False, but according to tests I should get [] in such cases